app/assets/javascripts/mumuki_laboratory/application/editors.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * This module allows to read and write the current editor's contents
+ * regardless if it is an standard editor or a custom editor
+ */
+mumuki.editors = {
```

Comment:
You will notice that I din't use a module-lambda here, just because it was not necessary. This is a direct consequence of building more cohesive module objects - only one module per file - using ES6 constructs, and clearly separating the initialization code - which is run only once - from the loading - `mumuki.load` - code  - which is loaded on page load and after turbolinks load event. 

I think we should tend to favor this construct when just a bunch of related functions are required, since it is less cumbersome.

Reply:
Ok, didn't see this one. Never mind my two comments related to the lambda function